controllers/blog.js

```diff
@@ -0,0 +1,65 @@
+const articleModel = require('../models/blog')
```

Comment:
Try use ES6 in your code. For example Instead of using require, you will use import. But if it is a module you are importing you will need to export it first.

Reply:
okay thanks